### PR TITLE
fix(ethereum): ensure randomBytes can return 0-255

### PR DESCRIPTION
**Pull request by davidmurdoch**
Previously, it returned only bytes 0 - 254.

### CODE REVIEW

**Review by mikeseese — APPROVED** [review]
LGTM

**Comment by mikeseese** [comment]
Do note this may change the behavior of `--deterministic` or `--seed`! It did for me (but it's ok because I haven't released)

**Comment by davidmurdoch** [comment]
> Do note this may change the behavior of --deterministic or --seed! It did for me (but it's ok because I haven't released)

BREAK ALL THE THINGS!

**Comment by davidmurdoch** [comment]
> Do note this may change the behavior of --deterministic or --seed! It did for me (but it's ok because I haven't released)

Ah, wait... crap. While this function doesn't matter for the start up accounts... there is another function that does have the same bug, even in version 2, and that WILL change things. hmmmmm

File: src/chains/ethereum/options/src/wallet-options.ts [hunk 1]
```diff
@@ -6,7 +6,13 @@ import { Definitions, DeterministicSeedPhrase } from "@ganache/options";
...
 const { alea } = seedrandom;
 
-function randomBytes(length: number, rng: () => number) {
+/**
+ * WARNING: to maintain compatibility with ganache v2 this RNG only generates
+ * numbers from 0-254 instead of 0-255! Hence the name, `notVeryRandomBytes`
+ * @param length
+ * @param rng
+ */
+function notVeryRandomBytes(length: number, rng: () => number) {
```
Comment (davidmurdoch) [inline src/chains/ethereum/options/src/wallet-options.ts:15]: @seesemichaelj can I get a 👍 if you approve of this new commit and message?

**Review by mikeseese — APPROVED** [review]

**Pull request merged**